documentation/contributing.md

```diff
@@ -22,4 +22,29 @@ conform to the
 1. Core migrations should be committed in their `goose fix`ed form. To do this, run `make version_migrations` which
 converts timestamped migrations to migrations versioned by an incremented integer.
 
+## Diagrams
+- Diagrams were created with [draw.io](draw.io).
+- To update a diagram:
+  1. Go to [draw.io](draw.io).
+  1. Click on *File > Open from* and choose the location of the diagram you want to update.
+  1. Once open in draw.io, you may update it.
+  1. Export the diagram to this repository's directory and add commit it.
```

Comment:
How does this work if it's stored as a `.png` in the project? :thinking:

Reply:
Do you mean like how are we able to update it one it's already been saved as a `.png` file? That's a great question, and I'm actually not sure! I'd be interested to see if someone else could give it a whirl to see if it works for others, and not just a _works on my machine_ thing.

Reply:
I'm not sure I follow - if you replaced the existing `.png` in the project with another one and committed, I would expect that to be reflected as a change to the repo, no?

Reply:
@rmulhol I mean, if you open a .png in draw.io, is it editable?

Reply:
Yeah, for me it is editable when I open it in draw.io. Would someone else be able to give it a try too? Just to make sure it's not just editable for me because I created it.

Reply:
Huh, awesome! Should be cool then :ok_hand: